recipes/zlib/1.2.11/conanfile.py

```diff
@@ -48,7 +44,11 @@ def build(self):
 
     @property
     def _use_autotools(self):
+        if self.settings.os == "iOS":
+            return False # use a cmake toolchain .... or, find out the special CHOST settings zlib requires, but ...
         return self.settings.os == "Linux" or tools.is_apple_os(self.settings.os)
```

Comment:
Is there a difference between `iOS` and `apple_os` ?

Reply:
yes, apple_os includes OSX
I did not want to change the build on osx, since this had no problems, just the iOS build needed the change,

Reply:
Thank you for explaining!